seedDatabase.js

```diff
@@ -0,0 +1,5 @@
+import SeedDataScripts from "./scripts/Seed.js";
+import config from './configurations/Configurations.js'
+
+await config.connectToDatabase()
```

Comment:
Kan du ta bort **await** och se ifall det fortfarande går att köra? isådanfall ta bort await.